src/development/development.service.ts

```diff
@@ -20,16 +20,9 @@ export class DevelopmentService {
         }
         try {
             await seed();
-            // logs the user out,
-            // otherwise there will be errors since logged in user will have a new uuid
-            return res
-                .status(HttpStatus.OK)
-                .clearCookie("access_token")
-                .clearCookie("refresh_token")
-                .json({
-                    message:
-                        "Database reseed successful. You are logged out. Please log in again.",
-                });
+            return res.status(HttpStatus.OK).json({
+                message: "Database reseed successful",
```

Comment:
how come all the clear cookie and logged out messages are removed? We will need to clear the cookies (logout) otherwise it may cause errors since the cookies don't match the values in the newly seeded database